### DISCUSSION TITLE
Will ccmenu2 be added into homebrew?

### DISCUSSION THREAD

**Discussion by flniu** [erikdoe/ccmenu2#19] [Q&A] [answered]
Current version of ccmenu in homebrew is 15.0.

**Comment by iloveitaly** [comment, marked as answer]
https://github.com/iloveitaly/homebrew-tap/blob/main/Casks/ccmenu2.rb

It can't be added to the Root homebrew because it's not popular enough yet.

**Reply by flniu** [reply]
Thanks!

`brew install iloveitaly/tap/ccmenu2` is convenient enough.

**Comment by erikdoe** [comment]
To be honest, I have no idea how CCMenu ended up in Homebrew. I can see this cask here:

https://github.com/Homebrew/homebrew-cask/blob/0a378013bc509b72c14391db6d6000e63dde09e9/Casks/c/ccmenu.rb

But how would I go about updating it? The actual change would be straight forward as the file by @iloveitaly demonstrates.
